### DISCUSSION TITLE
Is the precession of the orbit of Mercury accounted for?

### DISCUSSION THREAD

**Discussion by issantos** [Stellarium/stellarium#2053] [General]
The perihelion of Mercury shifts over time due to relativistic effects. Is this accounted for in Stellarium?

**Comment by gzotti** [comment]
You can be very sure about this. Stellarium replays simulation results from VSOP87, DE430, DE431, DE440 or DE441.

**Reply by AndiPangerangHasanuddin** [reply]
I placed DE43x and DE44x into _C:\Users\Andi Pangerang\AppData\Roaming\Stellarium\ephem_ folder
also into _C:\Program Files\Stellarium\ephem_ and _C:\Program Files (x86)\Stellarium\ephem_ folders respectively

<img width="686" alt="ephem stella_3" src="https://github.com/user-attachments/assets/cbaac541-ba1c-4598-9d22-0c87a3f46645">
<img width="691" alt="ephem stella_2" src="https://github.com/user-attachments/assets/94251d51-4e9e-4a0e-bfae-fb1caa23312f">
<img width="1494" alt="ephem stella_1" src="https://github.com/user-attachments/assets/b817abe6-4451-4a86-8852-f72b963178f4">

<img width="535" alt="ephem stella" src="https://github.com/user-attachments/assets/a0bb18c6-d50f-4ac2-a847-43fce56070d4">


why it doesn't appear?

**Reply by alex-w** [reply]
Stellarium expect to see files like `linux_p1550p2650.440` in `ephem` directory as it described in User Guide - not BSP!

**Reply by gzotti** [reply]
@AndiPangerangHasanuddin your question is off-topic in this thread.
